packages/public/camera-web/src/Camera/hooks/useUserMedia.ts

```diff
@@ -202,10 +220,36 @@ export function useUserMedia(constraints: MediaStreamConstraints): UserMediaResu
           stream.removeEventListener('inactive', onStreamInactive);
           stream.getTracks().forEach((track) => track.stop());
         }
-        const str = await navigator.mediaDevices.getUserMedia(constraints);
+        let str = await navigator.mediaDevices.getUserMedia(constraints);
+        const devices = await navigator.mediaDevices.enumerateDevices();
+        if (constraints) {
+          const videoConstraints = {
+            ...(constraints.video as MediaTrackConstraints),
+            deviceId: {
+              exact: devices
+                .filter(
+                  (device) =>
+                    device.kind === 'videoinput' &&
+                    !device.label.includes('Wide') &&
+                    !device.label.includes('Telephoto') &&
+                    !device.label.includes('Triple') &&
+                    !device.label.includes('Dual') &&
+                    !device.label.includes('Ultra'),
+                )
```

Comment:
for better readability, I suggest you create a filter function in the file called `isValidCamera`